docs/admin/usermgmt/README.md

```diff
@@ -4,13 +4,61 @@ tags:
   - user management
 ---
 
-## This content is in progress
+# Local User Management
 
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
+In this document we will cover managing people in the Jans Server's LDAP Directory, Jans CLI / TUI and using SCIM.
 
-## Have questions in the meantime?
+=== "Manage Data using DB Browser"
```

Comment:
Janssen Server doesn't use LDAP as it's default backend. Hence the documentation content has to be tuned accordingly. For instance, in this document, we should have content that talk about relational DB (like MySql which is our default persistence in PROD)